config/rbac/namespace_bindings/remote_namespace_role.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Same here, if the granted resources is namespaced, we better define a `Role` instead of `ClusterRole`, any specific reason choose `ClusterRole` with `RoleBinding`?